frontend/package.json

```diff
@@ -128,6 +128,7 @@
     "@emotion/jest": "^11.10.5",
     "@loaders.gl/images": "^3.2.13",
     "@testing-library/dom": "^8.19.0",
+    "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^12.1.4",
```

Comment:
Think it is worth adding this addtl. `devDependency` to add many semantic custom jest matchers ([info](https://www.npmjs.com/package/@testing-library/jest-dom))